GC: An error occurred (MissingContentMD5) when calling the DeleteObjects operation: Missing required header for this request: Content-Md5.

```
[2025-01-22 13:41:29,060][ERROR] GC: Exception during garbage collection
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/karton/system/system.py", line 213, in gc_collect
    self.gc_collect_resources()
  File "/usr/local/lib/python3.11/site-packages/karton/system/system.py", line 91, in gc_collect_resources
    self.backend.remove_objects(karton_bucket, resources_to_remove)
  File "/usr/local/lib/python3.11/site-packages/karton/core/backend.py", line 1006, in remove_objects
    self.s3.delete_objects(Bucket=bucket, Delete={"Objects": delete_objects})
  File "/usr/local/lib/python3.11/site-packages/botocore/client.py", line 569, in _api_call
    return self._make_api_call(operation_name, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/botocore/client.py", line 1023, in _make_api_call
    raise error_class(parsed_response, operation_name)
botocore.exceptions.ClientError: An error occurred (MissingContentMD5) when calling the DeleteObjects operation: Missing required header for this request: Content-Md5.
```

Seems related with https://github.com/boto/botocore/issues/2308

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,3 +1,4 @@
 redis
-boto3
 orjson
+# https://github.com/boto/boto3/issues/4392
+boto3<1.36.0

